src/Transport/GpsConfigurationResolver.php

```diff
@@ -19,7 +19,7 @@ final class GpsConfigurationResolver implements GpsConfigurationResolverInterfac
         self::BOOL_NORMALIZER_KEY => ['enableMessageOrdering', 'retainAckedMessages', 'enableExactlyOnceDelivery'],
     ];
     private const NORMALIZABLE_SUBSCRIPTION_PULL_OPTIONS = [
-        self::INT_NORMALIZER_KEY => ['maxMessages'],
+        self::INT_NORMALIZER_KEY => ['maxMessages', 'timeoutMillis'],
```

Comment:
My bad, I probably accidentally reverted it locally, but wouldn't be `setAllowedTypes()` sufficient and easier way?

Reply:
`setAllowedTypes` just checks the type. The gps-messenger-bundle also supports configuration via DSN (a URL-like string). After parsing the DSN, all values are treated as strings. To prevent unnecessary errors, we need to convert them to integers (or boolean) where applicable.
After the conversion, we can check the type again, but it will probably not be necessary.